addons/mrp/views/mrp_production_views.xml

```diff
@@ -50,6 +46,21 @@
             </field>
         </record>
 
+        <record id="mrp_production_tree_view_inherit_with_button" model="ir.ui.view">
+        <field name="name">mrp.production.tree.view.inherit</field>
+        <field name="model">mrp.production</field>
+        <field name="inherit_id" ref="mrp.mrp_production_tree_view"/>
+        <field name="mode">primary</field>
+        <field name="priority">100</field>
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="inside">
+                <button name="button_plan" type="object" string="Plan"/>
+                <button name="do_unreserve" type="object" string="Unreserve"/>
+                <button name="action_cancel" type="object" string="Cancel"/>
+            </xpath>
+        </field>
+    </record>
```

Comment:
you can add them directly into the basic production view

Reply:
nevermind I just checked the spec again

Reply:
@Whenrow we remove from basic production view because we need to hide these buttons on any pop-up of many2one of production